dask/dataframe/methods.py

```diff
@@ -398,6 +398,7 @@ def concat(
     join="outer",
     uniform=False,
     filter_warning=True,
+    ignore_order=False,
     ignore_index=False,
     **kwargs
```

Comment:
Just a note, it would be better to add new arguments near the end closer to `**kwargs`. Otherwise this causes breakage for downstream libraries

cc @kkraus14 (for vis)

Reply:
@jakirkham Thanks for the feedback. You are correct however it seems the `ignore_index` parameter is not used at all in the `concat_pandas` function. Should I open a PR changing the order of the parameters or removing the `ignore_index` parameter?

Reply:
Yeah I think that would be a good idea. Thanks Daniel! 😀

Reply:
We picked this up in PR ( https://github.com/dask/dask/pull/7500 ). Hope that is ok. Please take a look if you have a moment